hack/test/e2e.sh

```diff
@@ -22,12 +22,10 @@ mkdir -p "${TMP}"
 # Talos
 
 export TALOSCONFIG="${TMP}/talosconfig"
-export TALOS_VERSION=v0.9
 
 # Kubernetes
 
 export KUBECONFIG="${TMP}/kubeconfig"
-export K8S_VERSION=1.20.5
```

Comment:
`TALOS_VERSION` and `K8S_VERSION` are not referenced anywhere. Do we need them?

Reply:
They are referenced from the cluster template which is downloaded from another repository